CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.10.2)
 project(BSplineGradient)
 
 if(NOT ITK_SOURCE_DIR)
-  find_package(ITK REQUIRED)
```

Comment:
It is better to not put this version number here. It prevents building this remote module externally against an older version of ITK (e.g. 5.0, I don't know whether it would build against anything older).